mod_ci/controllers.py

```diff
@@ -905,6 +907,29 @@ def blocked_users():
             g.db.add(blocked_user)
             g.db.commit()
             flash('User blocked successfully.')
+            # Remove any queued pull request from blocked user
+            gh = GitHub(access_token=g.github['bot_token'])
+            repository = gh.repos(g.github['repository_owner'])(g.github['repository'])
+            try:
+                # Getting all pull requests by blocked user on the repo
+                pulls = requests.get('https://api.github.com/repos/ccextractor/sample-platform/pulls/head={username}'
```

Comment:
This url is not correct.

If possible, I'd prefer using the repository defined above. There's probably a possiblity to fetch it through there instead.